.github/workflows/publish-charms.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
           if [[ "${BRANCH}" == "main" ]]; then
             echo "track=latest" >> "$GITHUB_OUTPUT"
             echo "risk=edge" >> "$GITHUB_OUTPUT"
-          elif [[ "${BRANCH}" =~ "^release-[0-9]+\.[0-9]+$" ]]; then
+          elif if [[ "${BRANCH}" =~ ^release-[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Is this correct? I haven't seen this before

Reply:
fixed...